### PR TITLE
Use `go mod tidy` before make

### PR DESCRIPTION
**Pull request by akkie**
I have the issue with go 1.16 that I cannot build dapr because of a lot of errors with the message:

```
missing go.sum entry for module providing package ...
```

Maybe it's because of this issue: https://github.com/golang/go/issues/44129

Executing `go mod tidy` fixed it for me.

### CODE REVIEW

File: docs/developing-component.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ go mod edit -replace github.com/dapr/components-contrib=../components-contrib
 4. Register your component in dapr [main.go](https://github.com/dapr/dapr/blob/d17e9243b308e830649b0bf3af5f6e84fd543baf/cmd/daprd/main.go#L153-L226)(e.g. binding)
 5. Build debuggable dapr binary
 ```bash
+go mod tidy
```
Comment (artursouza) [inline docs/developing-component.md:65]: Can you add this to the github workflow too? Similar to how it is done in dapr/dapr, build should fail if `go mod tidy` creates any change.

**Review by artursouza — CHANGES_REQUESTED** [review]

File: docs/developing-component.md (hunk 1 above)
Comment (akkie) [reply docs/developing-component.md:65]: I will have a look

**Comment by yaron2** [comment]
@akkie  ping

**Comment by akkie** [comment]
I have this on my todo list. I'll first finish the PR for the zeebe binding

**Comment by akkie** [comment]
@yaron2 @artursouza It seems that  `go mod tidy` will not work currently for the dapr master.

```
❯ go mod tidy
go: finding module for package gopkg.in/gorethink/gorethink.v4/ql2
go: finding module for package gopkg.in/fatih/pool.v2
go: finding module for package gopkg.in/alexcesaro/quotedprintable.v3
go: finding module for package github.com/dapr/dapr/pkg/logger
go: finding module for package gopkg.in/gorethink/gorethink.v4/encoding
go: finding module for package gopkg.in/gorethink/gorethink.v4/types
go: finding module for package gopkg.in/couchbase/gocbcore.v7
go: finding module for package gopkg.in/couchbaselabs/jsonx.v1
go: finding module for package github.com/baiyubin/aliyun-sts-go-sdk/sts
go: finding module for package goji.io
go: finding module for package goji.io/pat
go: finding module for package gopkg.in/gorethink/gorethink.v4/internal/integration/tests
go: finding module for package github.com/gavv/httpexpect
go: finding module for package github.com/kataras/go-errors
go: finding module for package github.com/kataras/go-serializer
go: finding module for package github.com/nats-io/nats-streaming-server/server
go: finding module for package github.com/dnaeon/go-vcr/cassette
go: finding module for package github.com/dnaeon/go-vcr/recorder
go: finding module for package gopkg.in/couchbaselabs/gojcbmock.v1
go: finding module for package gopkg.in/jcmturner/goidentity.v3
go: found gopkg.in/fatih/pool.v2 in gopkg.in/fatih/pool.v2 v2.0.0
go: found gopkg.in/gorethink/gorethink.v4/encoding in gopkg.in/gorethink/gorethink.v4 v4.1.0
go: found gopkg.in/gorethink/gorethink.v4/ql2 in gopkg.in/gorethink/gorethink.v4 v4.1.0
go: found gopkg.in/gorethink/gorethink.v4/types in gopkg.in/gorethink/gorethink.v4 v4.1.0
go: found gopkg.in/alexcesaro/quotedprintable.v3 in gopkg.in/alexcesaro/quotedprintable.v3 v3.0.0-20150716171945-2caba252f4dc
go: found gopkg.in/couchbase/gocbcore.v7 in gopkg.in/couchbase/gocbcore.v7 v7.1.18
go: found gopkg.in/couchbaselabs/gocbconnstr.v1 in gopkg.in/couchbaselabs/gocbconnstr.v1 v1.0.0-00010101000000-000000000000
go: found gopkg.in/couchbaselabs/jsonx.v1 in gopkg.in/couchbaselabs/jsonx.v1 v1.0.0
go: found github.com/baiyubin/aliyun-sts-go-sdk/sts in github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f
go: found goji.io in goji.io v2.0.2+incompatible
go: found goji.io/pat in goji.io v2.0.2+incompatible
go: found gopkg.in/gorethink/gorethink.v4/internal/integration/tests in gopkg.in/gorethink/gorethink.v4 v4.1.0
go: found github.com/gavv/httpexpect in github.com/gavv/httpexpect v2.0.0+incompatible
go: found github.com/kataras/go-errors in github.com/kataras/go-errors v0.0.3
go: found github.com/kataras/go-serializer in github.com/kataras/go-serializer v0.0.4
go: found github.com/nats-io/nats-streaming-server/server in github.com/nats-io/nats-streaming-server v0.21.2
go: found github.com/dnaeon/go-vcr/cassette in github.com/dnaeon/go-vcr v1.1.0
go: found github.com/dnaeon/go-vcr/recorder in github.com/dnaeon/go-vcr v1.1.0
go: found gopkg.in/couchbaselabs/gojcbmock.v1 in gopkg.in/couchbaselabs/gojcbmock.v1 v1.0.4
go: found gopkg.in/jcmturner/goidentity.v3 in gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding module for package github.com/yudai/gojsondiff
go: finding module for package github.com/dapr/dapr/pkg/logger
go: finding module for package github.com/imkira/go-interpol
go: finding module for package github.com/ajg/form
go: finding module for package github.com/yudai/gojsondiff/formatter
go: finding module for package github.com/xeipuuv/gojsonschema
go: finding module for package github.com/moul/http2curl
go: finding module for package github.com/yalp/jsonpath
go: finding module for package github.com/fatih/structs
go: finding module for package gopkg.in/kataras/go-serializer.v0/data
go: finding module for package gopkg.in/kataras/go-serializer.v0/json
go: finding module for package gopkg.in/kataras/go-serializer.v0/jsonp
go: finding module for package gopkg.in/kataras/go-serializer.v0/markdown
go: finding module for package gopkg.in/kataras/go-serializer.v0/text
go: finding module for package gopkg.in/kataras/go-serializer.v0/xml
go: found github.com/ajg/form in github.com/ajg/form v1.5.1
go: found github.com/fatih/structs in github.com/fatih/structs v1.1.0
go: found github.com/imkira/go-interpol in github.com/imkira/go-interpol v1.1.0
go: found github.com/moul/http2curl in github.com/moul/http2curl v1.0.0
go: found github.com/xeipuuv/gojsonschema in github.com/xeipuuv/gojsonschema v1.2.0
go: found github.com/yalp/jsonpath in github.com/yalp/jsonpath v0.0.0-20180802001716-5cc68e5049a0
go: found github.com/yudai/gojsondiff in github.com/yudai/gojsondiff v1.0.0
go: found github.com/yudai/gojsondiff/formatter in github.com/yudai/gojsondiff v1.0.0
go: found gopkg.in/kataras/go-serializer.v0/data in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: found gopkg.in/kataras/go-serializer.v0/json in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: found gopkg.in/kataras/go-serializer.v0/jsonp in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: found gopkg.in/kataras/go-serializer.v0/markdown in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: found gopkg.in/kataras/go-serializer.v0/text in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: found gopkg.in/kataras/go-serializer.v0/xml in gopkg.in/kataras/go-serializer.v0 v0.0.4
go: finding module for package github.com/yudai/golcs
go: finding module for package github.com/dapr/dapr/pkg/logger
go: found github.com/yudai/golcs in github.com/yudai/golcs v0.0.0-20170316035057-ecda9a501e82
go: finding module for package github.com/dapr/dapr/pkg/logger
github.com/dapr/dapr/cmd/daprd imports
        github.com/dapr/components-contrib/bindings/zeebe/command imports
        github.com/dapr/dapr/pkg/logger: no matching versions for query "latest"
```

The make fails with:
```
❯ make DEBUG=1 build
Build with debugger information
which: no controller-gen in (/c/Program Files/PowerShell/7:/c/Python39/Scripts:/c/Python39:/c/WINDOWS/system32:/c/WINDOWS:/c/WINDOWS/System32/Wbem:/c/WINDOWS/System32/WindowsPowerShell/v1.0:/c/WINDOWS/System32/OpenSSH:/c/Program Files/dotnet:/c/ProgramData/chocolatey/bin:/cmd:/mingw64/bin:/usr/bin:/c/Program Files/nodejs:/c/Program Files/Microsoft SQL Server/130/Tools/Binn:/c/Program Files/Microsoft SQL Server/Client SDK/ODBC/170/Tools/Binn:/c/Program Files (x86)/dotnet:/c/Program Files/PowerShell/7:/c/Program Files/Go/bin:/c/Program Files/Docker/Docker/resources/bin:/c/ProgramData/DockerDesktop/version-bin:/c/Users/Christian.Kaps/scoop/shims:/c/Users/Christian.Kaps/AppData/Local/Microsoft/WindowsApps:/c/Users/Christian.Kaps/.dotnet/tools:/c/Users/Christian.Kaps/AppData/Local/Programs/Microsoft VS Code/bin:/c/dapr:/c/Users/Christian.Kaps/.dapr/bin:/c/Users/Christian.Kaps/.dapr/bin:/c/Users/Christian.Kaps/AppData/Roaming/npm:/c/Users/Christian.Kaps/.dotnet/tools:/c/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/MSBuild/Current/Bin:/c/Users/Christian.Kaps/go/bin)
CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -gcflags="all=-N -l" -ldflags="-X github.com/dapr/dapr/pkg/version.commit=v1.0.0-rc.4-124-g590acf7-dirty -X github.com/dapr/dapr/pkg/version.version=edge" -o ./dist/windows_amd64/debug/daprd.exe ./cmd/daprd/;
..\components-contrib\bindings\zeebe\client.go:14:2: no required module provides package github.com/dapr/dapr/pkg/logger; to add it:
        go get github.com/dapr/dapr/pkg/logger
..\components-contrib\bindings\zeebe\command\complete_job.go:14:2: missing go.sum entry for module providing package github.com/zeebe-io/zeebe/clients/go/pkg/commands (imported by github.com/dapr/components-contrib/bindings/zeebe/command); to add:
        go get github.com/dapr/components-contrib/bindings/zeebe/command@v1.1.1
..\components-contrib\bindings\zeebe\jobworker\jobworker.go:21:2: missing go.sum entry for module providing package github.com/zeebe-io/zeebe/clients/go/pkg/entities (imported by github.com/dapr/components-contrib/bindings/zeebe/jobworker); to add:
        go get github.com/dapr/components-contrib/bindings/zeebe/jobworker@v1.1.1
..\components-contrib\bindings\zeebe\command\deploy_workflow.go:16:2: missing go.sum entry for module providing package github.com/zeebe-io/zeebe/clients/go/pkg/pb (imported by github.com/dapr/components-contrib/bindings/zeebe/command); to add:
        go get github.com/dapr/components-contrib/bindings/zeebe/command@v1.1.1
..\components-contrib\bindings\zeebe\jobworker\jobworker.go:22:2: missing go.sum entry for module providing package github.com/zeebe-io/zeebe/clients/go/pkg/worker (imported by github.com/dapr/components-contrib/bindings/zeebe/jobworker); to add:
        go get github.com/dapr/components-contrib/bindings/zeebe/jobworker@v1.1.1
..\components-contrib\bindings\zeebe\client.go:15:2: missing go.sum entry for module providing package github.com/zeebe-io/zeebe/clients/go/pkg/zbc (imported by github.com/dapr/components-contrib/bindings/zeebe/command); to add:
        go get github.com/dapr/components-contrib/bindings/zeebe/command@v1.1.1
..\..\..\..\pkg\mod\google.golang.org\api@v0.32.0\option\option.go:12:2: missing go.sum entry for module providing package golang.org/x/oauth2 (imported by github.com/dapr/components-contrib/middleware/http/oauth2); to add:
        go get github.com/dapr/components-contrib/middleware/http/oauth2@v1.1.1
..\components-contrib\middleware\http\oauth2clientcredentials\oauth2clientcredentials_middleware.go:23:2: missing go.sum entry for module providing package golang.org/x/oauth2/clientcredentials (imported by github.com/dapr/components-contrib/middleware/http/oauth2clientcredentials); to add:
        go get github.com/dapr/components-contrib/middleware/http/oauth2clientcredentials@v1.1.1
..\..\..\..\pkg\mod\google.golang.org\api@v0.32.0\option\credentials_go19.go:10:2: missing go.sum entry for module providing package golang.org/x/oauth2/google (imported by google.golang.org/api/option); to add:
        go get google.golang.org/api/option@v0.32.0
..\..\..\..\pkg\mod\google.golang.org\grpc@v1.34.0\credentials\oauth\oauth.go:30:2: missing go.sum entry for module providing package golang.org/x/oauth2/jwt (imported by google.golang.org/grpc/credentials/oauth); to add:
        go get google.golang.org/grpc/credentials/oauth@v1.34.0
..\..\..\..\pkg\mod\cloud.google.com\go\pubsub@v1.3.1\subscription.go:31:2: missing go.sum entry for module providing package golang.org/x/sync/errgroup (imported by cloud.google.com/go/pubsub); to add:
        go get cloud.google.com/go/pubsub@v1.3.1
..\..\..\..\pkg\mod\cloud.google.com\go\pubsub@v1.3.1\flow_controller.go:21:2: missing go.sum entry for module providing package golang.org/x/sync/semaphore (imported by cloud.google.com/go/pubsub); to add:
        go get cloud.google.com/go/pubsub@v1.3.1
..\..\..\..\pkg\mod\gopkg.in\couchbase\gocb.v1@v1.6.4\auth.go:4:2: no required module provides package gopkg.in/couchbase/gocbcore.v7; to add it:
        go get gopkg.in/couchbase/gocbcore.v7
..\..\..\..\pkg\mod\gopkg.in\couchbase\gocb.v1@v1.6.4\cluster.go:13:2: module gopkg.in/couchbaselabs/gocbconnstr.v1 provides package gopkg.in/couchbaselabs/gocbconnstr.v1 and is replaced but not required; to add it:
        go get gopkg.in/couchbaselabs/gocbconnstr.v1
..\..\..\..\pkg\mod\gopkg.in\couchbase\gocb.v1@v1.6.4\cluster_searchquery.go:11:2: no required module provides package gopkg.in/couchbaselabs/jsonx.v1; to add it:
        go get gopkg.in/couchbaselabs/jsonx.v1
..\..\..\..\pkg\mod\github.com\dancannon\gorethink@v4.0.0+incompatible\pool.go:10:2: no required module provides package gopkg.in/fatih/pool.v2; to add it:
        go get gopkg.in/fatih/pool.v2
..\..\..\..\pkg\mod\github.com\dancannon\gorethink@v4.0.0+incompatible\cursor.go:11:2: no required module provides package gopkg.in/gorethink/gorethink.v4/encoding; to add it:
        go get gopkg.in/gorethink/gorethink.v4/encoding
..\..\..\..\pkg\mod\github.com\dancannon\gorethink@v4.0.0+incompatible\connection.go:13:2: no required module provides package gopkg.in/gorethink/gorethink.v4/ql2; to add it:
        go get gopkg.in/gorethink/gorethink.v4/ql2
..\..\..\..\pkg\mod\github.com\dancannon\gorethink@v4.0.0+incompatible\pseudotypes.go:9:2: no required module provides package gopkg.in/gorethink/gorethink.v4/types; to add it:
        go get gopkg.in/gorethink/gorethink.v4/types
make: *** [Makefile:136: dist/windows_amd64/debug/daprd.exe] Error 1
```

Any suggestion?

**Comment by akkie** [comment]
This was an issue with move from the logger package from dapr/dapr to dapr/kit @pkedy mentioned here: https://github.com/dapr/components-contrib/pull/806#issuecomment-825035288

Now it builds

File: docs/developing-component.md (hunk 1 above)
Comment (artursouza) [reply docs/developing-component.md:65]: Any updates on this?

Comment (akkie) [reply docs/developing-component.md:65]: Implemented it based on dapr/dapr. Hope this fits

**Review by artursouza — APPROVED** [review]

**Pull request merged**